### DISCUSSION TITLE
Queries not working with remote Hbase and Phoenix connector

### DISCUSSION THREAD

**Discussion by wonderfulvamsi** [trinodb/trino#23400] [General]
I am running HBase in standalone mode on a linux server..
I was able to query it with Phoenix in it..
I want to query it from Trino using the Phoenix connector..
The queries are failing but this doesn't happen when i try to with HBase running in my local..
Can someone please help me out..

<img width="803" alt="Screenshot 2024-07-20 at 11 43 35 PM" src="https://github.com/user-attachments/assets/c3cefa04-fd7f-4747-b62d-2db96c176f4d">

**Comment by ebyhr** [comment]
- https://github.com/trinodb/trino/pull/25207
